packages/ui-react/src/containers/AccountModal/AccountModal.tsx

```diff
@@ -62,7 +94,7 @@ const AccountModal = () => {
   }, [viewParam, loading, isPublicView, user, toLogin]);
 
   const closeHandler = useEventCallback(() => {
-    navigate(removeMultipleQueryParams(location, ['u', 'message']));
+    navigate(createURL(location.pathname, { u: null, message: null }, location.search));
```

Comment:
If there where any other query params, such as `utm_source` they will also be removed here. Right?
Maybe we should explicitly remove the 'u' and 'message' params and keep all others, to be safe and how it previously was.

Reply:
But I keep these other params with `location.search` right?

Reply:
You're right! I have said nothing